codespell_lib/data/dictionary.txt

```diff
@@ -13551,6 +13680,7 @@ placeholde->placeholder
 placeholdes->placeholders
 placeholer->placeholder
 placeholers->placeholders
+placematt->placemat
```

Comment:
maybe add `placement` to?

Reply:
Removed

Reply:
wasn't removed (which the request wasn't for) or changed.